chart/compass/charts/director/templates/ext-svc-certificate-rotation-job.yaml

```diff
@@ -26,7 +26,7 @@ spec:
           restartPolicy: Never
           containers:
             - name: {{ .Values.global.extSvcCertConfiguration.rotationCronjob.containerName }}
-              image: bitnami/kubectl:1.22.4
+              image: bitnami/kubectl:1.24.11
```

Comment:
Added as there is ARM build for this image. In job are used only compatible commands so it is without risk.

Reply:
reverted